```
Run nosetests tests
.....E..
======================================================================
ERROR: test_basic_client_update_stats_multi (tests.transplant_class.<locals>.C)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/pystatsd/pystatsd/tests/client.py", line 83, in test_basic_client_update_stats_multi
    self.mock_socket.return_value.sendto.assert_call_any(
  File "/opt/hostedtoolcache/Python/3.8.1/x64/lib/python3.8/site-packages/mock/mock.py", line 637, in __getattr__
    raise AttributeError("Attributes cannot start with 'assert' "
AttributeError: Attributes cannot start with 'assert' or 'assret'

----------------------------------------------------------------------
Ran 8 tests in 0.052s
```

--- a/.github/workflows/lint_python.yml
+++ b/.github/workflows/lint_python.yml
@@ -6,6 +6,9 @@ on:
 jobs:
   lint_python:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.8]
     steps:
       - uses: actions/checkout@master
       - uses: actions/setup-python@master

--- a/.github/workflows/lint_python.yml
+++ b/.github/workflows/lint_python.yml
@@ -12,6 +12,8 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - uses: actions/setup-python@master
+        with:
+          python-version: ${{ matrix.python-version }}
       - run: pip install black codespell flake8 isort  # pytest
       - run: black . --diff || true
       - run: codespell --quiet-level=2 || true  # --ignore-words-list="" --skip=""

--- a/.github/workflows/lint_python.yml
+++ b/.github/workflows/lint_python.yml
@@ -14,8 +14,8 @@ jobs:
       - uses: actions/setup-python@master
         with:
           python-version: ${{ matrix.python-version }}
-      - run: pip install black codespell flake8 isort  # pytest
-      - run: black . --diff || true
+      - run: pip install codespell flake8 isort  # black pytest
+      #- run: black . --diff || true
       - run: codespell --quiet-level=2 || true  # --ignore-words-list="" --skip=""
       - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
       - run: isort --recursive . || true

